---
date: "2022-12-20"
draft: false
excerpt:
subtitle: ""
title: Peer-reviewed articles
weight: 1
---

1. Mollandin, F., Gilbert, H., Croiseau, P., and **Rau, A.** (2022) Accounting for overlapping annotations in genomic prediction models of complex traits. *BMC Bioinformatics*, 23:65. [:link:](https://dx.doi.org/10.1186/s12859-022-04914-5)[:page_facing_up:](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04914-5)[:arrows_counterclockwise:](https://www.researchsquare.com/article/rs-1366477/v1)[:computer:](https://github.com/fmollandin/BayesRCO)

1. Mazurier, M., Drouaud, J., Bahrman, N., **Rau, A.**, Lejeune-Hénaut, I., Delbreil, B., and Legrand, S. (2022) Integrated sRNA-seq and RNA-seq analyses reveal a microRNA regulation network involved in cold response in Pisum sativum L. *Genes*, 13:1119. [:link:](https://dx.doi.org/10.3390/genes13071119)[:page_facing_up:](https://www.mdpi.com/2073-4425/13/7/1119/pdf)

1. **Rau, A.**, Passet, B., Castille, J., Asset, A., Lecardonnel, J., Moroldo, M., Jaffrézic, F., Laloë, D., Moazami-Goudarzi, K., and Vilotte, J.-L. (2022) Potential genetic robustness of Prnp and Sprn double knockout mouse embryos towards ShRNA-lentiviral inoculation. *Veterinary Research*, 53:54. [:link:](https://dx.doi.org/10.1186/s13567-022-01075-4)[:arrows_counterclockwise:](https://www.biorxiv.org/content/10.1101/2021.10.22.465458v1)[:input_numbers:](https://data.inrae.fr/privateurl.xhtml?token=e7b8885d-9cef-43c9-ae98-06259ee84d44)

1. Cazals, A., **Rau, A.**, Estellé, J., Bruneau, N., Coville, J.-L., Menanteau, P., Rossignol, M.-N., Jardet, D., Bevilacqua, C., Bed'Hom, B., Velge, P., and Calenge, F. (2022) Comparative analysis of the caecal tonsil transcriptome in two chicken lines experimentally infected with Salmonella Enteritidis. *PLoS ONE*, 17(8): e0270012. [:link:](https://dx.doi.org/10.1371/journal.pone.0270012)

1. Cazals, A., Estellé, J., Bruneau, N., Coville, J.-L., Menanteau, P., Rossignol, M.-N., Jardet, D., Bevilacqua, C., **Rau, A.**, Bed’Hom, B., Velge, P., and Calenge, F. (2022) Differences in caecal microbiota composition and Salmonella carriage between experimentally infected inbred lines of chickens. *Genetics Selection Evolution*, 54:7. [:link:](https://dx.doi.org/10.1186/s12711-022-00699-6)[:page_facing_up:](https://gsejournal.biomedcentral.com/articles/10.1186/s12711-022-00699-6)[:arrows_counterclockwise:](https://www.researchsquare.com/article/rs-76645/v1)

1. **Rau, A.**, Manansala, R., Flister, M. J., Rui, H., Jaffrézic, F., Laloë, D., and Auer, P. L. (2022) Individualized multi-omic pathway deviation scores using multiple factor analysis. *Biostatistics*, 23(2):362-379. [:link:](https://dx.doi.org/10.1093/biostatistics/kxaa029)[:page_facing_up:](https://academic.oup.com/biostatistics/article/23/2/362/5882080?guestAccessKey=fad8d201-b50c-47e5-b4b9-326e5fe62567)[:arrows_counterclockwise:](https://www.biorxiv.org/content/10.1101/827022v2)[:input_numbers:](https://github.com/andreamrau/RMFRJLA_2019)[:computer:](https://bioconductor.org/packages/padma/)

1. **Rau, A.** (2021) Cooking up knowledge from big data using data science. *Frontiers in Young Minds*, 9:632923. [:link:](https://dx.doi.org/10.3389/frym.2021.632923)[:page_facing_up:](https://kids.frontiersin.org/article/10.3389/frym.2021.632923)[:arrows_counterclockwise:](https://dx.doi.org/10.3389/frym.2021.632923)

1. Mollandin, F., **Rau, A.**, and Croiseau, P. (2021) An evaluation of the predictive performance and mapping power of the BayesR model for genomic prediction. *G3*, jkab225. [:link:](https://dx.doi.org/10.1093/g3journal/jkab225)[:page_facing_up:](https://academic.oup.com/g3journal/advance-article/doi/10.1093/g3journal/jkab225/6317672?login=true)[:arrows_counterclockwise:](https://www.biorxiv.org/content/10.1101/2020.10.23.351700v1)[:input_numbers:](https://github.com/fmollandin/BayesR_Simulations)

1. Sellem, E., Marthey, S., **Rau, A.**, Jouneau, L., Bonnet, A., Le Danvic, C., Kiefer, H., Jammes, H., and Schibler, L. (2021) Dynamics of cattle sperm sncRNAs during maturation, from testis to ejaculated sperm. *Epigenetics and Chromatin*, 14:24. [:link:](https://dx.doi.org/10.1186/s13072-021-00397-5)[:page_facing_up:](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-021-00397-5)

1. Mach, N., Moroldo, M., **Rau, A.**, Lecardonnel, J., Le Moyec, L., Robert, C., and Barrey, E. (2021) Understanding the holobiont: crosstalk between gut microbiota and mitochondria during endurance. *Frontiers Molecular Biosciences*, 8:656204. [:link:](https://dx.doi.org/10.3389/fmolb.2021.656204)[:page_facing_up:](https://www.frontiersin.org/articles/10.3389/fmolb.2021.656204/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Molecular_Biosciences&id=656204)[:arrows_counterclockwise:](https://www.biorxiv.org/content/10.1101/2021.01.08.425889v1)

1. Devogel, N., Auer, P. L., Manansala, R., **Rau, A.**, and Wang, T. (2021) A unified linear mixed model for familial relatedness and population structure in genetic association studies. *Genetic Epidemiology*, 45(3): 305-315. [:link:](https://dx.doi.org/10.1002/gepi.22371)[:page_facing_up:](https://onlinelibrary.wiley.com/doi/10.1002/gepi.22371)

1. Cho, Y., **Rau, A.**, Reiner, A., Auer, P. L. (2020) Mendelian randomization analysis with survival outcomes. *Genetic Epidemiology*, 45(1): 16-23. [:link:](https://dx.doi.org/10.1002/gepi.22354)[:page_facing_up:](https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.22354)

1. Sellem, E., Marthey, S., **Rau, A.**, Jouneau, L., Bonnet, A., Perrier, J.-P., Fritz, S., Le Danvic, C. Boussaha, M., Kiefer, H., Jammes, H., Schiblier, L. (2020) A comprehensive overview of bull sperm-borne small non-coding RNAs and their diversity in six breeds. *Epigenetics and Chromatin*, 13:19. [:link:](https://dx.doi.org/10.1186/s13072-020-00340-0)[:page_facing_up:](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00340-0)

1. Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2020) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. *Annals of Applied Statistics*, 14:2, 752-767. [:link:](https://dx.doi.org/10.1214/19-AOAS1317)[:page_facing_up:](https://projecteuclid.org/euclid.aoas/1593449324)[:input_numbers:](https://github.com/andreamrau/maskmeans)

1. Jehl, F., Désert, C., Klopp, C., Brenet, M., **Rau, A.**, Leroux, S., Boutin, M., Muret, K., Blum, Y., Esquerré, D., Gourichon, D., Burlot, T., Collin, A., Pitel, F., Benani, A., Zerjal, T., Lagarrigue, S. (2019) Chicken adaptive response to low energy diet: main role of the hypothalamic lipid metabolism revealed by a phenotypic and multi-tissue transcriptomic approach. *BMC Genomics*, 20. [:link:](https://dx.doi.org/10.1186/s12864-019-6384-8)[:page_facing_up:](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6384-8)

1. Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., **Rau, A.**, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Multi-species annotation of transcriptome and chromatin structure in domesticated animals. *BMC Biology*, 18:48. [:link:](https://dx.doi.org/10.1186/s12915-019-0726-5)[:page_facing_up:](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5)[:arrows_counterclockwise:](https://www.biorxiv.org/content/early/2018/05/11/316091v2)

1. Dhara, S., **Rau, A.**, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. *Scientific Reports*, 9:14198. [:link:](https://dx.doi.org/10.1038/s41598-019-50485-6)[:page_facing_up:](https://www.nature.com/articles/s41598-019-50485-6)[:arrows_counterclockwise:](https://www.biorxiv.org/content/10.1101/638734v3)[:input_numbers:](https://github.com/andreamrau/OpticRegen_2019)

1. **Rau, A.**, Dhara, S., Udvadia, A., and Auer, P. (2019) Regeneration Rosetta: An interactive web application to explore regeneration-associated gene expression and chromatin accessibility. *G3: Genes|Genomes|Genetics*, 9(12): 3953-3959. [:link:](https://dx.doi.org/10.1534/g3.119.400729)[:page_facing_up:](https://www.g3journal.org/content/9/12/3953)[:input_numbers:](https://github.com/andreamrau/rosetta)

1. Plasterer, C., Tsaih, S.-W., Lemke, A., Schilling, R., Dwinell, M., **Rau, A.**, Auer, P., Rui, H., Flister, M.J. (2019) Identification of a rat mammary tumor risk locus that is syntenic with the commonly amplified 8q12.1 and 8q22.1 regions in human breast cancer patients. *G3: Genes|Genomes|Genetics*, 9(5): 1739-1743. [:link:](https://dx.doi.org/10.1534/g3.118.200873)[:page_facing_up:](https://www.g3journal.org/content/9/5/1739)

1. Ramayo-Caldas, Y., Zingaretti, L., Bernard, A., Estellé, J. Popova, M., Pons, N., Bellot, P., Mach, N., **Rau, A.**, Roume, H., Perez-Encisco, M., Faverdin, P., Edouard, N., Dusko, S., Morgavi, D.P. and Renand, G. (2019) Identification of rumen microbial biomarkers linked to methane emission in Holstein dairy cows. *Journal of Animal Breeding and Genetics*, 137:49-59. [:link:](https://dx.doi.org/10.1111/jbg.12427)[:page_facing_up:](https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbg.12427)

1. **Rau, A.**, Flister, M. J., Rui, H. and Livermore Auer, P. (2019) Exploring drivers of gene expression in The Cancer Genome Atlas. *Bioinformatics*, 35(1): 62-68. [:link:](https://dx.doi.org/10.1093/bioinformatics/bty551)[:page_facing_up:](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764)[:arrows_counterclockwise:](https://www.biorxiv.org/content/early/2018/05/02/227926)[:input_numbers:](https://github.com/andreamrau/EDGE-in-TCGA)

1. Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. *Journal of Applied Statistics*, 46(1):47-65. [:link:](https://dx.doi.org/10.1080/02664763.2018.1454894)[:page_facing_up:](https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20)[:arrows_counterclockwise:](https://arxiv.org/abs/1704.06150)[:computer:](https://bioconductor.org/packages/release/bioc/html/coseq.html)

1. **Rau, A.** and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. *Briefings in Bioinformatics*, bbw128. [:link:](https://dx.doi.org/10.1093/bib/bbw128)[:page_facing_up:](http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354)[:arrows_counterclockwise:](https://www.biorxiv.org/content/early/2016/07/24/065607)[:computer:](https://bioconductor.org/packages/release/bioc/html/coseq.html)

1. Verrier, E., Genet, C., Laloë, D., Jaffrézic, J., **Rau, A.**, Esquerre, D., Dechamp, N., Ciobataru, C., Hervet, C., Krieg, F., Quillet, E., Boudinot, P. (2018) Genetic and transcriptomic analyses provide new insights on the early antiviral response to VHSV in resistant and susceptible rainbow trout. *BMC Genomics*, 19:482. [:link:](https://dx.doi.org/10.1186/s12864-018-4860-1)[:page_facing_up:](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4860-1)

1. Maroilley, T., Berri, M., Lemonnier, G., Esquerré, D., Chevaleyre, C., Mélo, S., Meurens, F., Coville, J.L., Leplat, J.J, **Rau, A.**, Bed’hom, B., Vincent-Naulleau, S., Mercat, M.J., Billon, Y., Lepage, P., Rogel-Gaillard, C., and Estellé, J. (2018) Immunome differences between porcine ileal and jejunal Peyer’s patches revealed by global transcriptome sequencing of gut-associated lymphoid tissues. *Scientific Reports*, 8:9077. [:link:](https://dx.doi.org/10.1038/s41598-018-27019-7)[:page_facing_up:](https://www.nature.com/articles/s41598-018-27019-7)

1. Mondet, F., **Rau, A.**, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018) Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. *BMC Genomics*, 19:328. [:link:](https://dx.doi.org/10.1186/s12864-018-4668-z)[:page_facing_up:](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z)

1. He, B., Tjhung, K., Bennett, N., Chou, Y., **Rau, A.**, Huang, J., and Derda, R. (2018) Compositional bias in naïve and chemically-modified phage-displayed libraries uncovered by paired-end deep sequencing. *Scientific Reports*, 8:1214. [:link:](https://dx.doi.org/10.1038/s41598-018-19439-2)[:page_facing_up:](http://rdcu.be/E4Zi)

1. Monneret, G., Jaffrézic, F., **Rau, A.**, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. *PLoS One*, 12(3): e0171142. [:link:](https://dx.doi.org/10.1371/journal.pone.0171142)[:page_facing_up:](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142)[:input_numbers:](https://github.com/Monneret/MarginalCausality)

1. Sauvage, C., **Rau, A.**, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Glémin, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. *The Plant Journal*, 91(4):631-645. [:link:](https://dx.doi.org/10.1111/tpj.13592)[:page_facing_up:](http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract)

1. Rigaill, G., Balzergue, S., Brunaud, V., Blondet, E., **Rau, A.**, Rogier, O., Caius, J., Maugis-Rabusseau, C., Soubigou-Taconnat, L., Aubourg, S., Lurin, C., Martin-Magniette, M.-L., and Delannoy, E. (2016) Synthetic datasets for the identification of key ingredients for RNA-seq differential analysis. *Briefings in Bioinformatics*, 19(1):65-76. [:link:](https://dx.doi.org/10.1093/bib/bbw092)[:page_facing_up:](https://academic.oup.com/bib/article/doi/10.1093/bib/bbw092/2339782/Synthetic-data-sets-for-the-identification-of-key)

1. Gallopin, M., Celeux, G., Jaffrézic, F., **Rau, A.** (2015) A model selection criterion for model-based clustering of annotated gene expression data. *Statistical Applications in Genetics and Molecular Biology*, 14(5): 413-428. [:link:](https://dx.doi.org/10.1515/sagmb-2014-0095)[:page_facing_up:](https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml)[:input_numbers:](https://github.com/Gallopin/ICAL)

1. Monneret, G., Jaffrézic, F., **Rau, A.**, Nuel, G. (2015) Estimation d’effets causaux dans les réseaux de régulation génique : vers la grande dimension. *Revue d’intelligence artificielle*, 29(2): 205-227.[:page_facing_up:](http://ria.revuesonline.com/article.jsp?articleId=20894)[:input_numbers:](https://github.com/andreamrau/GBNcausal)

1. **Rau, A.**, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. *Bioinformatics*, 31(9): 1420-1427. [:link:](https://dx.doi.org/10.1093/bioinformatics/btu845)[:page_facing_up:](http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract)[:arrows_counterclockwise:](https://hal.archives-ouvertes.fr/hal-01108821)[:computer:](https://cran.r-project.org/web/packages/HTSCluster)

1. **Rau, A.**, Marot, G. and Jaffrézic, F. (2014) Differential meta-analysis of RNA-seq data from multiple studies. *BMC Bioinformatics*, 16:31. [:link:](https://dx.doi.org/10.1186/1471-2105-15-91)[:page_facing_up:](http://www.biomedcentral.com/1471-2105/15/91/abstract)[:computer:](https://cran.r-project.org/web/packages/metaRNASeq/index.html)

1. Endale Ahanda, M.-L., Zerjal, T., Dhorne-Pollet, S., **Rau, A.**, Cooksey, A., and Giuffra, E. (2014) Impact of the genetic background on the composition of the chicken plasma miRNome in response to a stress. *PLoS One*, 9(12): e114598. [:link:](https://dx.doi.org/10.1371/journal.pone.0114598)[:page_facing_up:](http://www.plosone.org/article/info/%3Adoi/%2F10.1371/%2Fjournal.pone.0114598)

1. Nuel, G., **Rau, A.**, and Jaffrézic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments.. *Quality Technology and Quantitative Management*, 11(1):23-37. [:link:](https://dx.doi.org/10.1080/16843703.2014.11673323)[:page_facing_up:](http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf)

1. **Rau, A.**, Jaffrézic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. *BMC Systems Biology*, 8:51. [:link:](https://dx.doi.org/10.1186/1752-0509-7-111)[:page_facing_up:](http://www.biomedcentral.com/1752-0509/7/111/abstract)[:input_numbers:](https://github.com/andreamrau/GBNcausal)

1. Gallopin, M. **Rau, A.**, and Jaffrézic, F. (2013) A hierarchical Poisson log-normal model for network inference from RNA sequencing data. *PLoS One*, 8(10): e77503. [:link:](https://dx.doi.org/10.1371/journal.pone.0077503)[:page_facing_up:](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503)

1. **Rau, A.**, Gallopin, M., Celeux, G., and Jaffrézic, F. (2013) Data-based filtering for replicated high-throughput transcriptome sequencing experiments. *Bioinformatics*, 29(17): 2146-2152. [:link:](https://dx.doi.org/10.1093/bioinformatics/btt350)[:page_facing_up:](http://bioinformatics.oxfordjournals.org/content/29/17/2146)[:computer:](http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html)

1. Dillies, M.-A., **Rau, A.**, Aubert, J., Hennequet-Antier, C., Jeanmougin, M., Servant, N., Keime, C., Marot, G., Castel, D., Estelle, J., Guernec, G., Jagla, B., Jouneau, L., Laloë, D., Le Gall, C., Schaëffer, B., Charif, D., Le Crom, S., Guedj, M., and Jaffrézic, F. (2013) A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. *Briefings in Bioinformatics*, 14(6): 671-683. [:link:](https://dx.doi.org/10.1093/bib/bbs046)[:page_facing_up:](http://bib.oxfordjournals.org/content/14/6/671.abstract?sid=20bf56d5-b2c7-4de2-867d-e533fec9db25)

1. Brenault, P., Lefevre, L. **Rau, A.**, Laloë, D., Pisoni, G., Moroni, P., Bevilacquia, C. and Martin, P. (2013) Contribution of mammary epithelial cells to the immune response during early stages of a bacterial infection to Staphylococcus aureus. *Veterinary Research*, 45:16. [:link:](https://dx.doi.org/10.1186/1297-9716-45-16)[:page_facing_up:](http://www.veterinaryresearch.org/content/45/1/16)

1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2012) Reverse engineering gene regulatory networks using approximate Bayesian computation. *Statistics and Computing*, 22: 1257-1271. [:link:](https://dx.doi.org/10.1007/s11222-011-9309-1)[:page_facing_up:](http://link.springer.com/article/10.1007/%2Fs11222-011-9309-1)[:arrows_counterclockwise:](https://arxiv.org/abs/1109.1402)

1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2010) An empirical Bayesian method for estimating biological networks from temporal microarray data. *Statistical Applications in Genetics and Molecular Biology*, 9(1): 9. [:link:](https://dx.doi.org/10.2202/1544-6115.1513)[:page_facing_up:](http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml)[:computer:](https://cran.r-project.org/web/packages/ebdbNet)

1. Furth, A., Mandrekar, S., Tan, A. **Rau, A.**, Felten, S., Ames, M. Adjei, A. Erlichman, C. and Reid, J. (2008) A limited sample model to predict area under the drug concentration curve for 17-(allylamino)-17-demethoxygeldanamycin and its active metabolite 17-(amino)-17-demethoxygeldanomycin. *Cancer Chemotherapy Pharmacology*, 61(1): 39-45. [:link:](https://dx.doi.org/10.1007/s00280-007-0443-6)

