<!DOCTYPE html>
<html lang="en" dir="ltr"><head>
  
                           
     


<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="generator" content="Hugo 0.88.1" />
<title>Peer-reviewed articles | Andrea Rau, PhD</title>


<meta property="twitter:site" content="@andreamrau">
<meta property="twitter:creator" content="@andreamrau">







  
    
  
<meta name="description" content="A modern, beautiful, and easily configurable blog theme for Hugo.">


<meta property="og:site_name" content="Andrea Rau, PhD">
<meta property="og:title" content="Peer-reviewed articles | Andrea Rau, PhD">
<meta property="og:description" content="A modern, beautiful, and easily configurable blog theme for Hugo." />
<meta property="og:type" content="page" />
<meta property="og:url" content="https://andreamrau.netlify.app/collection/writing/articles/" />
<meta property="og:locale" content="en">




    
        <meta property="og:image" content="https://andreamrau.netlify.app/img/papillons.jpg" >
        <meta property="twitter:card" content="summary_large_image">
        <meta name="twitter:image" content="https://andreamrau.netlify.app/img/papillons.jpg" >
    
  <meta itemprop="name" content="Peer-reviewed articles">
<meta itemprop="description" content="Rau, A. (2021) Cooking up knowledge from big data using data science. Frontiers in Young Minds, 9:632923. ðŸ“„ ðŸ”„ ðŸ—£ ðŸ”— ðŸ’»
  Mollandin, F., Rau, A., and Croiseau, P. (2021) An evaluation of the predictive performance and mapping power of the BayesR model for genomic prediction. G3, jkab225. ðŸ“„ ðŸ”„ ðŸ’»
  Sellem, E., Marthey, S., Rau, A., Jouneau, L., Bonnet, A., Le Danvic, C., Kiefer, H."><meta itemprop="datePublished" content="2021-01-01T00:00:00+00:00" />
<meta itemprop="dateModified" content="2021-01-01T00:00:00+00:00" />
<meta itemprop="wordCount" content="1487">
<meta itemprop="keywords" content="" />
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-133763862-2', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

  <!--[if IE]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon">
  <link rel="icon" href="/img/favicon.ico" type="image/x-icon">
  
  
  <link rel="stylesheet" href="/style.main.min.f72f5cc794fcc7b86cab520db152dc3ef55aea333df2e3496a4e1bac97849df2.css" integrity="sha256-9y9cx5T8x7hsq1INsVLcPvVa6jM98uNJak4brJeEnfI=" media="screen">
  
  
  <script src="/panelset.min.078a92db9bd3228df502db3d9e0453c3cf3d910abe3f8deca0ad196c7071ad41.js" type="text/javascript"></script>
  
  
  <script src="/main.min.1d7d7e679af4c4b8656e42f9579ef37a0da08882bedab865e1016e58bce5e19b.js" type="text/javascript"></script>
</head>
<body>
      <div class="grid-container single-series">
<header class="site-header pt4 pb2 mb4 bb b--transparent ph5 headroom z-max" role="banner">
  <nav class="site-nav db dt-l w-100" role="navigation">
    <a class="site-brand db dtc-l v-mid link no-underline w-100 w-33-l tc tl-l" href="https://andreamrau.netlify.app/" title="Home">
      <img src="/img/blogophonic-mark-dark.png" class="dib db-l h2 w-auto" alt="Andrea Rau, PhD">
    </a>
    <div class="site-links db dtc-l v-mid w-100 w-47-l tc tr-l mt3 mt0-l ttu tracked">
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/about/" title="About">About</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/project/" title="Projects">Projects</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/talk/" title="talks">Talks</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 active" href="/collection/" title="Research">Research</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/blog/" title="Blog">Blog</a>
      
      
    </div>
  </nav>
</header>


<main class="page-main ph4 pt4" role="main">
  <section class="page-content mw7 center">
    <article class="post-content pa0 pr3-l">
      <header class="post-header">
        <h4 class="breadcrumb pb5">

 
 
    
 
 
    
 
 
    
 

  
    
      <a href="/collection/">Research contributions</a> >
    


  
    
      <a href="/collection/writing/">Writing</a> >
    


  
    
      Peer-reviewed articles
    


</h4>




        
        <h1 class="f1 lh-solid measure-narrow mb3 fw4">Peer-reviewed articles</h1>
        
        
        
        
        
      
        
      </header>
      <section class="post-body pt5 pb4">
        <ol>
<li>
<p><strong>Rau, A.</strong> (2021) Cooking up knowledge from big data using data science. <em>Frontiers in Young Minds</em>, 9:632923.

<a href="https://kids.frontiersin.org/article/10.3389/frym.2021.632923" target="_blank" rel="noopener">ðŸ“„</a>

<a href="https://dx.doi.org/10.3389/frym.2021.632923" target="_blank" rel="noopener">ðŸ”„</a>
ðŸ—£
ðŸ”—
ðŸ’»</p>
</li>
<li>
<p>Mollandin, F., <strong>Rau, A.</strong>, and Croiseau, P. (2021) An evaluation of the predictive performance and mapping power of the BayesR model for genomic prediction. <em>G3</em>, jkab225.

<a href="https://academic.oup.com/g3journal/advance-article/doi/10.1093/g3journal/jkab225/6317672?login=true" target="_blank" rel="noopener">ðŸ“„</a>

<a href="https://www.biorxiv.org/content/10.1101/2020.10.23.351700v1" target="_blank" rel="noopener">ðŸ”„</a> 

<a href="https://github.com/fmollandin/BayesR_Simulations" target="_blank" rel="noopener">ðŸ’»</a></p>
</li>
<li>
<p>Sellem, E., Marthey, S., <strong>Rau, A.</strong>, Jouneau, L., Bonnet, A., Le Danvic, C., Kiefer, H., Jammes, H., and Schibler, L. (2021) Dynamics of cattle sperm sncRNAs during maturation, from testis to ejaculated sperm. <em>Epigenetics and Chromatin</em>, 14:24.
<a class="btn btn-primary btn-outline btn-xs"   href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-021-00397-5" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Mach, N., Moroldo, M., <strong>Rau, A.</strong>, Lecardonnel, J., Le Moyec, L., Robert, C., and Barrey, E. (2021) Understanding the holobiont: crosstalk between gut microbiota and mitochondria during endurance. <em>Frontiers Molecular Biosciences</em>, 8:656204.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03197210" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.frontiersin.org/articles/10.3389/fmolb.2021.656204/full?&utm_source=Email_to_authors_&utm_medium=Email&utm_content=T1_11.5e1_author&utm_campaign=Email_publication&field=&journalName=Frontiers_in_Molecular_Biosciences&id=656204" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/2021.01.08.425889v1" target="_blank" rel="noopener"> 
Preprint 
</a></p>
</li>
<li>
<p>Devogel, N., Auer, P. L., Manansala, R., <strong>Rau, A.</strong>, and Wang, T. (2020) A unified linear mixed model for familial relatedness and population structure in genetic association studies. <em>Genetic Epidemiology</em>, 45(3): 305-315.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03176147v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/10.1002/gepi.22371" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Cho, Y., <strong>Rau, A.</strong>, Reiner, A., Auer, P. L. (2020) Mendelian randomization analysis with survival outcomes. <em>Genetic Epidemiology</em>, 45(1): 16-23.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03155097v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.22354" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Sellem, E., Marthey, S., <strong>Rau, A.</strong>, Jouneau, L., Bonnet, A., Perrier, J.-P., Fritz, S., Le Danvic, C. Boussaha, M., Kiefer, H., Jammes, H., Schiblier, L. (2020) A comprehensive overview of bull sperm-borne small non-coding RNAs and their diversity in six breeds. <em>Epigenetics and Chromatin</em>, 13:19.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-03176141v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00340-0" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, Manansala, R., Flister, M. J., Rui, H., JaffrÃ©zic, F., LaloÃ«, D., and Auer, P. L. (2020) Individualized multi-omic pathway deviation scores using multiple factor analysis. <em>Biostatistics</em>, kxaa029.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02968360v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/biostatistics/advance-article/doi/10.1093/biostatistics/kxaa029/5882080?guestAccessKey=9a15cdfb-100e-4478-8b68-e0a1f9e495f4" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/827022v2" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/padma/" target="_blank" rel="noopener"> 
Software 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/RMFRJLA_2019" target="_blank" rel="noopener"> 
Code 
</a></p>
</li>
<li>
<p>Godichon-Baggioni, A., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2020) Multi-view cluster aggregation and splitting, with an application to multi-omic breast cancer data. <em>Annals of Applied Statistics</em>, 14:2, 752-767.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789493" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://projecteuclid.org/euclid.aoas/1593449324" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/maskmeans" target="_blank" rel="noopener"> 
Code 
</a></p>
</li>
<li>
<p>Jehl, F., DÃ©sert, C., Klopp, C., Brenet, M., <strong>Rau, A.</strong>, Leroux, S., Boutin, M., Muret, K., Blum, Y., EsquerrÃ©, D., Gourichon, D., Burlot, T., Collin, A., Pitel, F., Benani, A., Zerjal, T., Lagarrigue, S. (2019) Chicken adaptive response to low energy diet: main role of the hypothalamic lipid metabolism revealed by a phenotypic and multi-tissue transcriptomic approach. <em>BMC Genomics</em>, 20.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02463367v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6384-8" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Foissac, S., Djebali, S., Munyard, K., Villa-Vialaneix, N., <strong>Rau, A.</strong>, Muret, K., Esquerre, D., Zytnicki, M., Derrien, T., Bardou, P., Blanc, F., Cabau, C., Crisci, E., Dhorne-Pollet, S., Drouet, F., Gonzales, I., Goubil, A., Lacroix-Lamande, S., Laurent, F., Marthey, S., Marti-Marimon, M., Momal-Leisenring, R., Mompart, F., Quere, P., Robelin, D., San Cristobal, M., Tosser-Klopp, G., Vincent-Naulleau, S., Fabre, S., Pinard-Van der Laan, M.-H., Klopp, C., Tixier-Boichard, M., Acloque, H., Lagarrigue, S., Giuffra, E. (2019) Multi-species annotation of transcriptome and chromatin structure in domesticated animals. <em>BMC Biology</em>, 18:48.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02437599v1" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-019-0726-5" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/11/316091v2" target="_blank" rel="noopener"> 
Preprint 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="http://www.fragencode.org/" target="_blank" rel="noopener"> 
Website 
</a></p>
</li>
<li>
<p>Dhara, S., <strong>Rau, A.</strong>, Flister, M., Recka, N., Laiosa, M., Auer, P., and Udvadia, A. (2019) Cellular reprogramming for successful CNS axon regeneration is driven by a temporally changing cast of transcription factors. <em>Scientific Reports</em>, 9:14198.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02791772" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-019-50485-6" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/10.1101/638734v3" target="_blank" rel="noopener"> 
Preprint 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/OpticRegen_2019" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener"> 
Shiny app 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, Dhara, S., Udvadia, A., and Auer, P. (2019) Regeneration Rosetta: An interactive web application to explore regeneration-associated gene expression and chromatin accessibility. <em>G3: Genes|Genomes|Genetics</em>, 9(12): 3953-3959.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/hal-02789080" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/12/3953" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/rosetta" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/rosetta/" target="_blank" rel="noopener"> 
Shiny app 
</a></p>
</li>
<li>
<p>Plasterer, C., Tsaih, S.-W., Lemke, A., Schilling, R., Dwinell, M., <strong>Rau, A.</strong>, Auer, P., Rui, H., Flister, M.J. (2019) Identification of a rat mammary tumor risk locus that is syntenic with the commonly amplified 8q12.1 and 8q22.1 regions in human breast cancer patients. <em>G3: Genes|Genomes|Genetics</em>, 9(5): 1739-1743.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02626057" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.g3journal.org/content/9/5/1739" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Ramayo-Caldas, Y., Zingaretti, L., Bernard, A., EstellÃ©, J. Popova, M., Pons, N., Bellot, P., Mach, N., <strong>Rau, A.</strong>, Roume, H., Perez-Encisco, M., Faverdin, P., Edouard, N., Dusko, S., Morgavi, D.P. and Renand, G. (2019) Identification of rumen microbial biomarkers linked to methane emission in Holstein dairy cows. <em>Journal of Animal Breeding and Genetics</em>, 137:49-59.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02495627" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbg.12427" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, Flister, M. J., Rui, H. and Livermore Auer, P. (2019) Exploring drivers of gene expression in The Cancer Genome Atlas. <em>Bioinformatics</em>, 35(1): 62-68.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02625326" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty551/5047764" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2018/05/02/227926" target="_blank" rel="noopener"> 
Preprint 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/EDGE-in-TCGA" target="_blank" rel="noopener"> 
Code 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ls-shiny-prod.uwm.edu/edge_in_tcga/" target="_blank" rel="noopener"> 
Shiny app 
</a></p>
</li>
<li>
<p>Godichon-Baggioni, A., Maugis-Rabusseau, C. and <strong>Rau, A.</strong> (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. <em>Journal of Applied Statistics</em>, 46(1):47-65.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02789493" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://arxiv.org/abs/1704.06150" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener"> 
Software 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong> and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. <em>Briefings in Bioinformatics</em>, bbw128.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02624483" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.biorxiv.org/content/early/2016/07/24/065607" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bioconductor.org/packages/release/bioc/html/coseq.html" target="_blank" rel="noopener"> 
Software 
</a></p>
</li>
<li>
<p>Verrier, E., Genet, C., LaloÃ«, D., JaffrÃ©zic, J., <strong>Rau, A.</strong>, Esquerre, D., Dechamp, N., Ciobataru, C., Hervet, C., Krieg, F., Quillet, E., Boudinot, P. (2018) Genetic and transcriptomic analyses provide new insights on the early antiviral response to VHSV in resistant and susceptible rainbow trout. <em>BMC Genomics</em>, 19:482.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02444518" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4860-1" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Maroilley, T., Berri, M., Lemonnier, G., EsquerrÃ©, D., Chevaleyre, C., MÃ©lo, S., Meurens, F., Coville, J.L., Leplat, J.J, <strong>Rau, A.</strong>, Bedâ€™hom, B., Vincent-Naulleau, S., Mercat, M.J., Billon, Y., Lepage, P., Rogel-Gaillard, C., and EstellÃ©, J. (2018) Immunome differences between porcine ileal and jejunal Peyerâ€™s patches revealed by global transcriptome sequencing of gut-associated lymphoid tissues. <em>Scientific Reports</em>, 8:9077.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02356215" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.nature.com/articles/s41598-018-27019-7" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Mondet, F., <strong>Rau, A.</strong>, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018) Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. <em>BMC Genomics</em>, 19:328.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/inserm-02063466" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>He, B., Tjhung, K., Bennett, N., Chou, Y., <strong>Rau, A.</strong>, Huang, J., and Derda, R. (2018) Compositional bias in naÃ¯ve and chemically-modified phage-displayed libraries uncovered by paired-end deep sequencing. <em>Scientific Reports</em>, 8:1214.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02623801" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://rdcu.be/E4Zi" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Monneret, G., JaffrÃ©zic, F., <strong>Rau, A.</strong>, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. <em>PLoS One</em>, 12(3): e0171142.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01504607" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/Monneret/MarginalCausality" target="_blank" rel="noopener"> 
Code 
</a></p>
</li>
<li>
<p>Sauvage, C., <strong>Rau, A.</strong>, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., GlÃ©min, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. <em>The Plant Journal</em>, 91(4):631-645.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01594937" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Rigaill, G., Balzergue, S., Brunaud, V., Blondet, E., <strong>Rau, A.</strong>, Rogier, O., Caius, J., Maugis-Rabusseau, C., Soubigou-Taconnat, L., Aubourg, S., Lurin, C., Martin-Magniette, M.-L., and Delannoy, E. (2016) Synthetic datasets for the identification of key ingredients for RNA-seq differential analysis. <em>Briefings in Bioinformatics</em>, 19(1):65-76.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01595551" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://academic.oup.com/bib/article/doi/10.1093/bib/bbw092/2339782/Synthetic-data-sets-for-the-identification-of-key" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Gallopin, M., Celeux, G., JaffrÃ©zic, F., <strong>Rau, A.</strong> (2015) A model selection criterion for model-based clustering of annotated gene expression data. <em>Statistical Applications in Genetics and Molecular Biology</em>, 14(5): 413-428.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01255908" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/Gallopin/ICAL" target="_blank" rel="noopener"> 
Code 
</a></p>
</li>
<li>
<p>Monneret, G., JaffrÃ©zic, F., <strong>Rau, A.</strong>, Nuel, G. (2015) Estimation dâ€™effets causaux dans les rÃ©seaux de rÃ©gulation gÃ©nique : vers la grande dimension. <em>Revue dâ€™intelligence artificielle</em>, 29(2): 205-227.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194142" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://ria.revuesonline.com/article.jsp?articleId=20894" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener"> 
Code 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. <em>Bioinformatics</em>, 31(9): 1420-1427.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01108821" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://hal.archives-ouvertes.fr/hal-01108821" target="_blank" rel="noopener"> 
Preprint 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/HTSCluster" target="_blank" rel="noopener"> 
Software 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, Marot, G. and JaffrÃ©zic, F. (2014) Differential meta-analysis of RNA-seq data from multiple studies. <em>BMC Bioinformatics</em>, 16:31.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00978902" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.biomedcentral.com/1471-2105/15/91/abstract" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/metaRNASeq/index.html" target="_blank" rel="noopener"> 
Software 
</a></p>
</li>
<li>
<p>Endale Ahanda, M.-L., Zerjal, T., Dhorne-Pollet, S., <strong>Rau, A.</strong>, Cooksey, A., and Giuffra, E. (2014) Impact of the genetic background on the composition of the chicken plasma miRNome in response to a stress. <em>PLoS One</em>, 9(12): e114598.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194096" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.plosone.org/article/info/%3Adoi/%2F10.1371/%2Fjournal.pone.0114598" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Nuel, G., <strong>Rau, A.</strong>, and JaffrÃ©zic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments.. <em>Quality Technology and Quantitative Management</em>, 11(1):23-37.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01194064" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, JaffrÃ©zic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. <em>BMC Systems Biology</em>, 8:51.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193793" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.biomedcentral.com/1752-0509/7/111/abstract" target="_blank" rel="noopener"> 
PDF 
</a>     <a class="btn btn-primary btn-outline btn-xs"   href="https://github.com/andreamrau/GBNcausal" target="_blank" rel="noopener"> 
Code 
</a></p>
</li>
<li>
<p>Gallopin, M. <strong>Rau, A.</strong>, and JaffrÃ©zic, F. (2013) A hierarchical Poisson log-normal model for network inference from RNA sequencing data. <em>PLoS One</em>, 8(10): e77503.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01004715" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, Gallopin, M., Celeux, G., and JaffrÃ©zic, F. (2013) Data-based filtering for replicated high-throughput transcriptome sequencing experiments. <em>Bioinformatics</em>, 29(17): 2146-2152.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00927025" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bioinformatics.oxfordjournals.org/content/29/17/2146" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="http://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html" target="_blank" rel="noopener"> 
Software 
</a></p>
</li>
<li>
<p>Dillies, M.-A., <strong>Rau, A.</strong>, Aubert, J., Hennequet-Antier, C., Jeanmougin, M., Servant, N., Keime, C., Marot, G., Castel, D., Estelle, J., Guernec, G., Jagla, B., Jouneau, L., LaloÃ«, D., Le Gall, C., SchaÃ«ffer, B., Charif, D., Le Crom, S., Guedj, M., and JaffrÃ©zic, F. (2013) A comprehensive evaluation of normalization methods for Illumina high-throughput RNA sequencing data analysis. <em>Briefings in Bioinformatics</em>, 14(6): 671-683.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-00782486" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://bib.oxfordjournals.org/content/14/6/671.abstract?sid=20bf56d5-b2c7-4de2-867d-e533fec9db25" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p>Brenault, P., Lefevre, L. <strong>Rau, A.</strong>, LaloÃ«, D., Pisoni, G., Moroni, P., Bevilacquia, C. and Martin, P. (2013) Contribution of mammary epithelial cells to the immune response during early stages of a bacterial infection to Staphylococcus aureus. <em>Veterinary Research</em>, 45:16.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193839" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.veterinaryresearch.org/content/45/1/16" target="_blank" rel="noopener"> 
PDF 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, JaffrÃ©zic, F., Foulley, J.-L., and Doerge, R. W. (2012) Reverse engineering gene regulatory networks using approximate Bayesian computation. <em>Statistics and Computing</em>, 22: 1257-1271.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01019032" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://link.springer.com/article/10.1007/%2Fs11222-011-9309-1" target="_blank" rel="noopener"> 
PDF 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="https://arxiv.org/abs/1109.1402" target="_blank" rel="noopener"> 
Preprint 
</a></p>
</li>
<li>
<p><strong>Rau, A.</strong>, JaffrÃ©zic, F., Foulley, J.-L., and Doerge, R. W. (2010) An empirical Bayesian method for estimating biological networks from temporal microarray data. <em>Statistical Applications in Genetics and Molecular Biology</em>, 9(1): 9.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-01193424" target="_blank" rel="noopener"> 
HAL 
</a>   <a class="btn btn-primary btn-outline btn-xs"   href="http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml" target="_blank" rel="noopener"> 
PDF 
</a>    <a class="btn btn-primary btn-outline btn-xs"   href="https://cran.r-project.org/web/packages/ebdbNet" target="_blank" rel="noopener"> 
Software 
</a></p>
</li>
<li>
<p>Furth, A., Mandrekar, S., Tan, A. <strong>Rau, A.</strong>, Felten, S., Ames, M. Adjei, A. Erlichman, C. and Reid, J. (2008) A limited sample model to predict area under the drug concentration curve for 17-(allylamino)-17-demethoxygeldanamycin and its active metabolite 17-(amino)-17-demethoxygeldanomycin. <em>Cancer Chemotherapy Pharmacology</em>, 61(1): 39-45.
<a class="btn btn-primary btn-outline btn-xs"   href="https://hal.inrae.fr/view/index/identifiant/hal-02664062" target="_blank" rel="noopener"> 
HAL 
</a></p>
</li>
</ol>

      </section>
      <footer class="post-footer">
        <div class="post-pagination dt w-100 mt4 mb2">
  
  
  
  
    <a class="next dtc pl2 tr v-top fw6"
    href="https://andreamrau.netlify.app/collection/writing/books/">Books, book chapters &amp; dissertations &rarr;</a>
  
</div>

      </footer>
    </article>
    
  </section>
</main>

<aside class="page-sidebar" role="complementary">
                         
 











  <img src="/collection/featured-sidebar.jpg" class="db ma0">

                         
 



<div class="flex items-start sticky ph4 pb4 flex-row">
  <div class="w-two-thirds w-50-l ph0">
    <h2 class="mv3 f5 fw7 ttu tracked">
      <a class="no-underline dim" href="/collection/">Outline</a></h2>
    <nav id="SectionTableOfContents" aria-label="SectionTableOfContents">
        <ul>
        
          
          
          
          <details open class="active">
            <summary class="active" hugo-nav="/collection/writing/"><a href="https://andreamrau.netlify.app/collection/writing/">Writing</a></summary>
            
              
              <li class="active" hugo-nav="/collection/writing/articles/"><a href="https://andreamrau.netlify.app/collection/writing/articles/">Peer-reviewed articles</a></li>
            
              
              <li class="" hugo-nav="/collection/writing/books/"><a href="https://andreamrau.netlify.app/collection/writing/books/">Books, book chapters &amp; dissertations</a></li>
            
              
              <li class="" hugo-nav="/collection/writing/preprints/"><a href="https://andreamrau.netlify.app/collection/writing/preprints/">Preprints</a></li>
            
          </details>
          
          </li>
        
          
          
          
          <details  class="">
            <summary class="" hugo-nav="/collection/coding/"><a href="https://andreamrau.netlify.app/collection/coding/">Coding</a></summary>
            
              
              <li class="" hugo-nav="/collection/coding/packages/"><a href="https://andreamrau.netlify.app/collection/coding/packages/">R software packages</a></li>
            
              
              <li class="" hugo-nav="/collection/coding/shiny/"><a href="https://andreamrau.netlify.app/collection/coding/shiny/">Shiny apps</a></li>
            
          </details>
          
          </li>
        
          
          
          
          <details  class="">
            <summary class="" hugo-nav="/collection/advising/"><a href="https://andreamrau.netlify.app/collection/advising/">Advising</a></summary>
            
              
              <li class="" hugo-nav="/collection/advising/phd/"><a href="https://andreamrau.netlify.app/collection/advising/phd/">PhD &#43; Postdoc supervision</a></li>
            
              
              <li class="" hugo-nav="/collection/advising/masters/"><a href="https://andreamrau.netlify.app/collection/advising/masters/">Master&#39;s interns &#43; short-term contract supervision</a></li>
            
              
              <li class="" hugo-nav="/collection/advising/committees/"><a href="https://andreamrau.netlify.app/collection/advising/committees/">Committee membership</a></li>
            
          </details>
          
          </li>
        
        </ul>
    </nav>
  </div>
  
</div>

</aside>

<footer class="site-footer pv4 bt b--transparent ph5" role="contentinfo">
  <nav class="db dt-l w-100">
    <p class="site-copyright f7 db dtc-l v-mid w-100 w-33-l tc tl-l pv2 pv0-l mv0 lh-copy">
      &copy; 2022 RStudio, Anywhere
      <span class="middot-divider"></span>
      Made with <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/hugo-apero/" rel="dct:source">Hugo ApÃ©ro</a></span>.
      <br />
      
Based on <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/formspree/blogophonic-hugo" rel="dct:source">Blogophonic</a></span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://formspree.io" property="cc:attributionName" rel="cc:attributionURL">Formspree</a>.
    </p>
    
    <div class="site-social-links db dtc-l v-mid w-100 w-33-l tc pv2 pv0-l mv0">
      <div class="social-icon-links" aria-hidden="true">
  
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="envelope" >
      <i class="fa fa-envelope fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="twitter" >
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="google-scholar" >
      <i class="ai ai-google-scholar fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="github" >
      <i class="fab fa-github fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="orcid" >
      <i class="ai ai-orcid fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="researchgate" >
      <i class="ai ai-researchgate fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="linkedin" >
      <i class="fab fa-linkedin fa-lg fa-fw"></i>
    </a>
  
</div>

    </div>
    
    <div class="site-links f6 db dtc-l v-mid w-100 w-67-l tc tr-l pv2 pv0-l mv0">
      
      <a class="dib pv1 ph2 link" href="/license/" title="License">License</a>
      
    </div>
  </nav>
  
    <script>

    var i, text, code, codes = document.getElementsByTagName('code');
    for (let i = 0; i < codes.length;) {
      code = codes[i];
      if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
        text = code.textContent;
        if (/^\$[^$]/.test(text) && /[^$]\$$/.test(text)) {
          text = text.replace(/^\$/, '\\(').replace(/\$$/, '\\)');
          code.textContent = text;
        }
        if (/^\\\((.|\s)+\\\)$/.test(text) ||
            /^\\\[(.|\s)+\\\]$/.test(text) ||
            /^\$(.|\s)+\$$/.test(text) ||
            /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
          code.outerHTML = code.innerHTML;  
          continue;
        }
      }
      i++;
    }
</script>

  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>



    
  
  
</footer>

      </div>
    </body>
</html>
