---
author: Andrea Rau
categories:
- projects
date: "2023-02-10"
draft: false
excerpt: Co-expression analysis for RNA sequencing data
layout: single
subtitle:
tags:
- hugo-site
title: RNA-seq co-expression
---

### Related software

1. **coseq**: Co-expression analysis of sequencing data [:link:](https://bioconductor.org/packages/coseq/)
1. **ICAL**: Model selection for model based clustering of annotated data [:link:](https://github.com/Gallopin/ICAL)
1. **HTSCluster**: Clustering high-throughput sequencing data with Poisson mixture models [:link:](http://cran.r-project.org/web/packages/HTSCluster)

### Related publications

1. Godichon-Baggioni, A., Maugis-Rabusseau, C. and **Rau, A.** (2018) Clustering transformed compositional data using K-means, with applications in gene expression and bicycle sharing system data. *Journal of Applied Statistics*, 46(1):47-65. [:link:](https://dx.doi.org/10.1080/02664763.2018.1454894)[:page_facing_up:](https://www.tandfonline.com/doi/abs/10.1080/02664763.2018.1454894?journalCode=cjas20)[:arrows_counterclockwise:](https://arxiv.org/abs/1704.06150)[:computer:](https://bioconductor.org/packages/release/bioc/html/coseq.html)

1. **Rau, A.** and Maugis-Rabusseau, C. (2018) Transformation and model choice for RNA-seq co-expression analysis. *Briefings in Bioinformatics*, bbw128. [:link:](https://dx.doi.org/10.1093/bib/bbw128)[:page_facing_up:](http://academic.oup.com//bib/article/doi/10.1093/bib/bbw128/2870509/Transformation-and-model-choice-for-RNAseq?guestAccessKey=83aebf08-b670-4da2-b227-dc5f5acab354)[:arrows_counterclockwise:](https://www.biorxiv.org/content/early/2016/07/24/065607)[:computer:](https://bioconductor.org/packages/release/bioc/html/coseq.html)

1. Mondet, F., **Rau, A.**, Klopp, C., Rohmer, M. Severac, D., Le Conte, Y., and Alaux, C. (2018) Transcriptome profiling of the honeybee parasite Varroa destructor provides new biological insights into the mite adult life cycle. *BMC Genomics*, 19:328. [:link:](https://dx.doi.org/10.1186/s12864-018-4668-z)[:page_facing_up:](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4668-z)

1. Sauvage, C., **Rau, A.**, Aichholz, C., Chadoeuf, J., Sarah, G., Ruiz, M., Santoni, S., Causse, M., David, J., Glémin, S. (2017) Domestication rewired gene expression and nucleotide diversity patterns in tomato. *The Plant Journal*, 91(4):631-645. [:link:](https://dx.doi.org/10.1111/tpj.13592)[:page_facing_up:](http://onlinelibrary.wiley.com/doi/10.1111/tpj.13592/abstract)

1. Gallopin, M., Celeux, G., Jaffrézic, F., **Rau, A.** (2015) A model selection criterion for model-based clustering of annotated gene expression data. *Statistical Applications in Genetics and Molecular Biology*, 14(5): 413-428. [:link:](https://dx.doi.org/10.1515/sagmb-2014-0095)[:page_facing_up:](https://www.degruyter.com/view/j/sagmb.2015.14.issue-5/sagmb-2014-0095/sagmb-2014-0095.xml)[:input_numbers:](https://github.com/Gallopin/ICAL)

1. **Rau, A.**, Maugis-Rabusseau, C., Martin-Magniette, M.-L., Celeux, G. (2015) Co-expression analysis of high-throughput transcriptome sequencing data with Poisson mixture models. *Bioinformatics*, 31(9): 1420-1427. [:link:](https://dx.doi.org/10.1093/bioinformatics/btu845)[:page_facing_up:](http://bioinformatics.oxfordjournals.org/content/early/2015/01/05/bioinformatics.btu845.abstract)[:arrows_counterclockwise:](https://hal.archives-ouvertes.fr/hal-01108821)[:computer:](https://cran.r-project.org/web/packages/HTSCluster)

