<!DOCTYPE html>
<html lang="en" dir="ltr"><head>
  
                           
     


<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="generator" content="Hugo 0.88.1" />
<title>Research Talks | Andrea Rau, PhD</title>


<meta property="twitter:site" content="@andreamrau">
<meta property="twitter:creator" content="@andreamrau">







  
    
  
<meta name="description" content="Check out these recent and upcoming talks in conferences, workshops, and seminars. 
">


<meta property="og:site_name" content="Andrea Rau, PhD">
<meta property="og:title" content="Research Talks | Andrea Rau, PhD">
<meta property="og:description" content="Check out these recent and upcoming talks in conferences, workshops, and seminars. 
" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://andreamrau.netlify.app/talk/" />
<meta property="og:locale" content="en">




    
        <meta property="og:image" content="https://andreamrau.netlify.app/talk/sidebar-listing.jpg" >
        <meta property="twitter:card" content="summary_large_image">
        <meta name="twitter:image" content="https://andreamrau.netlify.app/talk/sidebar-listing.jpg" >
    
    
  <meta itemprop="name" content="Research Talks">
<meta itemprop="description" content="Check out these recent and upcoming talks in conferences, workshops, and seminars. 
">
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-133763862-2', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

  <!--[if IE]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <link rel="shortcut icon" href="/img/sigle.ico" type="image/x-icon">
  <link rel="icon" href="/img/sigle.ico" type="image/x-icon">
  
    <link rel="alternate" type="application/rss+xml" href="https://andreamrau.netlify.app/talk/index.xml" title="Andrea Rau, PhD" />
  
  
  <link rel="stylesheet" href="/style.main.min.f72f5cc794fcc7b86cab520db152dc3ef55aea333df2e3496a4e1bac97849df2.css" integrity="sha256-9y9cx5T8x7hsq1INsVLcPvVa6jM98uNJak4brJeEnfI=" media="screen">
  
  
  <script src="/panelset.min.078a92db9bd3228df502db3d9e0453c3cf3d910abe3f8deca0ad196c7071ad41.js" type="text/javascript"></script>
  
  
  <script src="/main.min.87b8bc3638f41ea1fd41f332acc64b4dd3be1dc21037b1984d4c12dccf1f876f.js" type="text/javascript"></script>
</head>
<body>
      <div class="grid-container list">
<header class="site-header pt4 pb2 mb4 bb b--transparent ph5 headroom z-max" role="banner">
  <nav class="site-nav db dt-l w-100" role="navigation">
    <a class="site-brand db dtc-l v-mid link no-underline w-100 w-33-l tc tl-l" href="https://andreamrau.netlify.app/" title="Home">
      <img src="/img/Logo-INRAE_Transparent.png" class="dib db-l h2 w-auto" alt="Andrea Rau, PhD">
    </a>
    <div class="site-links db dtc-l v-mid w-100 w-47-l tc tr-l mt3 mt0-l ttu tracked">
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/about/" title="About">About</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/project/" title="Projects">Projects</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 active" href="/talk/" title="talks">Talks</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/collection/" title="Research">Research</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/blog/" title="Blog">Blog</a>
      
      
    </div>
  </nav>
</header>

<main class="page-main pa4" role="main">
  <section class="blog-intro mw7 center mb5">
    <h2 class="f2 f1-ns lh-solid tc center fw4">Research Talks</h2>
    <p class="f6 f5-ns measure lh-copy tc center">Check out these recent and upcoming talks in conferences, workshops, and seminars.</p>
    <p class="f7 measure lh-copy tc center i"></p>
  </section>
  <section class="blog-content mw7 center">
    
    
    
    
    <section class="group-content pb3 pb4-ns bt">
      <h5 class="f4 fw7 tracked ttu lh-title mv3">2025</h5>
      
          
                                     
     
     

<article class="pt4 w-100">

  
  
  
  <div class="flex flex-column flex-row-ns mb3 mb4-l">
    <figure class="mr3-ns ma0 mb2 mb0-l w-100 w-30-ns">
      <a href="/talk/2025-01-06-jouy-en-josas/" class="db">
        
      </a>
    </figure>
    <div class="flex flex-column w-100 w-70-ns pl3-ns">
      <header class="w-100">
        <h1 class="f4 mv1 lh-title underline"><a href="/talk/2025-01-06-jouy-en-josas/" class="db">Tackling data integration challenges &ndash; Integrated transcriptomic and epigenetic differential analysis with idiffomix</a></h1>
        
      </header>
      
      
      <footer class="w-100">
      
      <p class="f7 dib mv1">





  
  
      January 6, 2025
  
  
</p>
      
      
        <span class="middot-divider"></span>
        <p class="f7 dib mv1">Jouy en Josas</p>
      
      <br />
      
      
    </footer>
    </div>
  </div>

  
  

</article>
      
    </section>    
    
    
    <section class="group-content pb3 pb4-ns bt">
      <h5 class="f4 fw7 tracked ttu lh-title mv3">2024</h5>
      
          
                                     
     
     

<article class="pt4 w-100">

  
  
  
  <div class="flex flex-column flex-row-ns mb3 mb4-l">
    <figure class="mr3-ns ma0 mb2 mb0-l w-100 w-30-ns">
      <a href="/talk/2024-12-03-aussois/" class="db">
        
      </a>
    </figure>
    <div class="flex flex-column w-100 w-70-ns pl3-ns">
      <header class="w-100">
        <h1 class="f4 mv1 lh-title underline"><a href="/talk/2024-12-03-aussois/" class="db">(Invited keynote) From complexity to clarity &ndash; Tackling the challenges of multi-omic integration</a></h1>
        
      </header>
      
      <p class="f6 lh-copy mv2 flex-auto measure-wide">The increasing availability and affordability of high-throughput sequencing technologies have enabled the generation of large-scale multi-omic data, greatly enhancing our understanding of complex biological systems across hierarchical molecular levels. A great deal of attention has been devoted to developing integrative methods that can fully leverage these multifaceted data, despite numerous statistical challenges such as small sample sizes, high dimensionality, heterogeneous measures, missing data, and complex interdependencies within and between omic layers. To date, many multi-omic integrative approaches have been proposed, reflecting the diversity of omics combinations, definitions of inter-omic anchors, and analysis objectives. In this talk, I will provide an overview of some commonly used methods for multi-omics integration, and I will introduce one of our own recent contributions in this field &ndash; idiffomix, a joint mixture model for integrated differential analyses of paired transcriptomic and methylation data. I will conclude by discussing some future opportunities and challenges in integrative multi-omics research.</p>
      <footer class="w-100">
      
      <p class="f7 dib mv1">





  
  
      December 3, 2024
  
  
</p>
      
      
        <span class="middot-divider"></span>
        <p class="f7 dib mv1">Aussois</p>
      
      <br />
      
      
    </footer>
    </div>
  </div>

  
  

</article>
      
          
                                     
     
     

<article class="pt4 w-100">

  
  
  
  <div class="flex flex-column flex-row-ns mb3 mb4-l">
    <figure class="mr3-ns ma0 mb2 mb0-l w-100 w-30-ns">
      <a href="/talk/2024-11-26-marseille/" class="db">
        
      </a>
    </figure>
    <div class="flex flex-column w-100 w-70-ns pl3-ns">
      <header class="w-100">
        <h1 class="f4 mv1 lh-title underline"><a href="/talk/2024-11-26-marseille/" class="db">(Invited talk) Accounting for overlapping functional annotations as biological priors in genomic prediction models of complex traits</a></h1>
        
      </header>
      
      <p class="f6 lh-copy mv2 flex-auto measure-wide">It is now widespread to build whole-genome regression models using genomic data to predict complex traits in a wide range of fields, including farm animal and plant breeding and human genetics. Functional genomic annotations, such as the accessibility of chromatin or methylation status in target tissues at relevant developmental stages, have the potential to provide valuable insight into the position and effect size of causal genetic variants underlying complex traits. In the H2020 GENE-SWitCH project, we aimed to develop and validate Bayesian models able to more fully leverage such complex functional annotations for improved accuracy and interpretability of genomic predictions in the pig and poultry breeding sectors. To this end, we defined and implemented a flexible framework for genomic prediction called BayesRCO to simultaneously take advantage of the availability of multiple functional genomic annotations. In this talk, I’ll describe the intuition behind our proposed model and discuss some of our key take-away messages from early use cases.</p>
      <footer class="w-100">
      
      <p class="f7 dib mv1">





  
  
      November 26, 2024
  
  
</p>
      
      
        <span class="middot-divider"></span>
        <p class="f7 dib mv1">Marseille</p>
      
      <br />
      
      
    </footer>
    </div>
  </div>

  
  

</article>
      
          
                                     
     
     

<article class="pt4 w-100">

  
  
  
  <div class="flex flex-column flex-row-ns mb3 mb4-l">
    <figure class="mr3-ns ma0 mb2 mb0-l w-100 w-30-ns">
      <a href="/talk/2024-03-29-virtual/" class="db">
        
      </a>
    </figure>
    <div class="flex flex-column w-100 w-70-ns pl3-ns">
      <header class="w-100">
        <h1 class="f4 mv1 lh-title underline"><a href="/talk/2024-03-29-virtual/" class="db">Using biological priors in genomic prediction models &ndash; QQOQCCP?</a></h1>
        
      </header>
      
      
      <footer class="w-100">
      
      <p class="f7 dib mv1">





  
  
      March 29, 2024
  
  
</p>
      
      
        <span class="middot-divider"></span>
        <p class="f7 dib mv1">virtual</p>
      
      <br />
      
      
    </footer>
    </div>
  </div>

  
  

</article>
      
          
                                     
     
     

<article class="pt4 w-100">

  
  
  
  <div class="flex flex-column flex-row-ns mb3 mb4-l">
    <figure class="mr3-ns ma0 mb2 mb0-l w-100 w-30-ns">
      <a href="/talk/2024-03-07-dublin-ireland/" class="db">
        
      </a>
    </figure>
    <div class="flex flex-column w-100 w-70-ns pl3-ns">
      <header class="w-100">
        <h1 class="f4 mv1 lh-title underline"><a href="/talk/2024-03-07-dublin-ireland/" class="db">(Invited talk) Accounting for overlapping functional annotations as biological priors in Bayesian genomic prediction models of complex traits</a></h1>
        
      </header>
      
      <p class="f6 lh-copy mv2 flex-auto measure-wide">It is now widespread to build whole-genome regression models using genomic data to predict complex traits in a wide range of fields, including farm animal and plant breeding and human genetics. Functional genomic annotations, such as the accessibility of chromatin or methylation status in relevant tissues, have the potential to provide valuable insight into the position and effect size of causal genetic variants underlying complex traits. In the H2020 GENE-SWitCH project, we aimed to develop and validate Bayesian models able to fully leverage such complex functional annotations for improved accuracy and interpretability of genomic predictions in the pig and poultry breeding sectors. To this end, we defined and implemented a flexible framework for genomic prediction called BayesRCO to simultaneously take advantage of the availability of multiple functional genomic annotations. In this talk, I’ll describe the intuition behind our proposed model and discuss some of our key take-away messages from early use cases.</p>
      <footer class="w-100">
      
      <p class="f7 dib mv1">





  
  
      March 7, 2024
  
  
</p>
      
      
        <span class="middot-divider"></span>
        <p class="f7 dib mv1">Dublin, Ireland</p>
      
      <br />
      
      
    </footer>
    </div>
  </div>

  
  

</article>
      
    </section>    
    
    



<div class="list-pagination w-100">
  <ul class="paginate list pl0 tc dt w-100 ma0">
    <li class="dtc w-25 tl v-mid">
    
      <span class="previous pa2 db fw6">&larr; Newer</span>
    
    </li>
    <li class="dtc w-50 v-mid">
      <span class="page_number fw6 f7 ttu tracked">1 of 5</span>
    </li>
    <li class="dtc w-25 tr v-mid">
    
      <a href="/talk/page/2/" class="next pa2 db fw6" aria-label="Next">Older &rarr;</a>
    
    </li>
  </ul>
</div>


  </section>
</main>
<footer class="site-footer pv4 bt b--transparent ph5" role="contentinfo">
  <nav class="db dt-l w-100">
    <p class="site-copyright f7 db dtc-l v-mid w-100 w-33-l tc tl-l pv2 pv0-l mv0 lh-copy">
      &copy; 2025 Andrea Rau, INRAE
      <span class="middot-divider"></span>
      Made with <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/hugo-apero/" rel="dct:source">Hugo Apéro</a></span>.
      <br />
      
Based on <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/formspree/blogophonic-hugo" rel="dct:source">Blogophonic</a></span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://formspree.io" property="cc:attributionName" rel="cc:attributionURL">Formspree</a>.
    </p>
    
    <div class="site-social-links db dtc-l v-mid w-100 w-33-l tc pv2 pv0-l mv0">
      <div class="social-icon-links" aria-hidden="true">
  
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="envelope" >
      <i class="fa fa-envelope fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="twitter" >
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="google-scholar" >
      <i class="ai ai-google-scholar fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="github" >
      <i class="fab fa-github fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="orcid" >
      <i class="ai ai-orcid fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="researchgate" >
      <i class="ai ai-researchgate fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="/" title="linkedin" >
      <i class="fab fa-linkedin fa-lg fa-fw"></i>
    </a>
  
</div>

    </div>
    
    <div class="site-links f6 db dtc-l v-mid w-100 w-67-l tc tr-l pv2 pv0-l mv0">
      
      <a class="dib pv1 ph2 link" href="/license/" title="License">License</a>
      
    </div>
  </nav>
  
    <script>

    var i, text, code, codes = document.getElementsByTagName('code');
    for (let i = 0; i < codes.length;) {
      code = codes[i];
      if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
        text = code.textContent;
        if (/^\$[^$]/.test(text) && /[^$]\$$/.test(text)) {
          text = text.replace(/^\$/, '\\(').replace(/\$$/, '\\)');
          code.textContent = text;
        }
        if (/^\\\((.|\s)+\\\)$/.test(text) ||
            /^\\\[(.|\s)+\\\]$/.test(text) ||
            /^\$(.|\s)+\$$/.test(text) ||
            /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
          code.outerHTML = code.innerHTML;  
          continue;
        }
      }
      i++;
    }
</script>

  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>



    
  
  
</footer>

      </div>
    </body>
</html>
