---
author: Andrea Rau
categories:
- projects
date: "2023-02-10"
draft: false
excerpt: Systems biology approaches to identify dependency networks.
layout: single
subtitle:
tags:
- hugo-site
title: Network inference
---

### Related software

1. **ebdbNet**: Empirical Bayes estimation for dynamic Bayesian networks [:link:](http://cran.r-project.org/web/packages/ebdbNet)

### Related publications

1. Monneret, G., Jaffrézic, F., **Rau, A.**, Zerjal, T. and Nuel, G. (2017) Identification of marginal causal relationships in gene networks from observational and interventional expression data. *PLoS One*, 12(3): e0171142. [:link:](https://dx.doi.org/10.1371/journal.pone.0171142)[:page_facing_up:](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0171142)[:input_numbers:](https://github.com/Monneret/MarginalCausality)

1. Monneret, G., Jaffrézic, F., **Rau, A.**, Nuel, G. (2015) Estimation d’effets causaux dans les réseaux de régulation génique : vers la grande dimension. *Revue d’intelligence artificielle*, 29(2): 205-227.[:page_facing_up:](http://ria.revuesonline.com/article.jsp?articleId=20894)[:input_numbers:](https://github.com/andreamrau/GBNcausal)

1. Nuel, G., **Rau, A.**, and Jaffrézic, F. (2013) Using pairwise ordering preferences to estimate causal effects in gene expression from a mixture of observational and intervention experiments.. *Quality Technology and Quantitative Management*, 11(1):23-37. [:link:](https://dx.doi.org/10.1080/16843703.2014.11673323)[:page_facing_up:](http://web.it.nctu.edu.tw/~qtqm/qtqmpapers/2014V11N1/2014V11N1_F2.pdf)

1. **Rau, A.**, Jaffrézic, F., and Nuel, G. (2013) Joint estimation of causal effects from observational and intervention gene expression data. *BMC Systems Biology*, 8:51. [:link:](https://dx.doi.org/10.1186/1752-0509-7-111)[:page_facing_up:](http://www.biomedcentral.com/1752-0509/7/111/abstract)[:input_numbers:](https://github.com/andreamrau/GBNcausal)

1. Gallopin, M. **Rau, A.**, and Jaffrézic, F. (2013) A hierarchical Poisson log-normal model for network inference from RNA sequencing data. *PLoS One*, 8(10): e77503. [:link:](https://dx.doi.org/10.1371/journal.pone.0077503)[:page_facing_up:](http://www.plosone.org/article/info:doi/10.1371/journal.pone.0077503)

1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2012) Reverse engineering gene regulatory networks using approximate Bayesian computation. *Statistics and Computing*, 22: 1257-1271. [:link:](https://dx.doi.org/10.1007/s11222-011-9309-1)[:page_facing_up:](http://link.springer.com/article/10.1007/%2Fs11222-011-9309-1)[:arrows_counterclockwise:](https://arxiv.org/abs/1109.1402)

1. **Rau, A.**, Jaffrézic, F., Foulley, J.-L., and Doerge, R. W. (2010) An empirical Bayesian method for estimating biological networks from temporal microarray data. *Statistical Applications in Genetics and Molecular Biology*, 9(1): 9. [:link:](https://dx.doi.org/10.2202/1544-6115.1513)[:page_facing_up:](http://www.degruyter.com/view/j/sagmb.2010.9.1/sagmb.2010.9.1.1513/sagmb.2010.9.1.1513.xml)[:computer:](https://cran.r-project.org/web/packages/ebdbNet)

